### ISSUE TITLE
Need help with livestreamer internet issue. please help

### ISSUE THREAD

**Issue by ghost** [chrippa/livestreamer#727]
I am sorry if this is the wrong place to make this thread. I am new to this site. Also let me start off by saying that livestreamer is a gift from god/the person who created it because it solved a lot of problems for me.

I have been using livestreamer since early november and I have had random bugs here and there, the worst and most recent was last night where I was watching a stream and my internet was fine, but vlc went grey and then the stream started to loop, I restarted it and it was fine, I have also gotten random error codes etc... But I have been dealing with a weird problem for a little over 2 weeks now and I believe (I am willing to be wrong) I have narrowed it down to livestreamer. 

First the problem. At random times during the day when I will be using the internet or playing a game such as hearthstone or magic online, my internet will cut out for a split second and then come back as if nothing happened. My router also stays the same, no resets, it doesn't turn off etc... I am always watching a stream using livestreamer when the internet cuts.

I have tried different things to solve this issue. Changing my router did nothing, scanning my computer for viruses and found nothing. I removed my comodo firewall and still the problem persisted. I then made a firewall exception ( I hope I correctly did) for livestreamer in windows firewall and still these things happen. 

The reason I believe it is livestreamer is because it is always on when I am on my computer and watching a stream and even by itself these things can happen although the cut outs happen a little more randomly. It also happened today when I was doing nothing but watching a stream and I was on facebook, I looked at the cmd prompt and saw an error code 11004. When I am playing hearthstone by itself, nothing happens, when I am surfing the internet and going about my business without livestreamer being used, nothing happens. I am hoping someone here can help me with this problem as it is very disturbing to me and it really screws up things when gaming etc... If anyone here also has idea's on how to check what it could be, please let me know. I have searched a lot through git hub but no one seems to have this problem but me, much like the grey screen issue. I found someone who experienced it, but they had a wifi issue and I am not on wifi.

Also if it helps, I am on a wired Ethernet connection. Also drivers up to date and the wire is working properly and is not broken. Again thank you to whoever can help me. I have my phone and my laptop on wifi, they never lose internet connection and so I know the modem and router are good. I have run disc checks, error checks for my computer and every time, the report says nothing is broken and I am all good to go.

**Comment by strider112** [comment]
can no one shed any light on this issue? Granted its a pretty unusual one I would suspect.

**Comment by chrippa** [comment]
The only thing I can think of is that perhaps your connection has problems keeping a stable link when it is put under strain over a long period of time. Does running the stream at low or mobile quality help for example?

**Comment by strider112** [comment]
First, thank you very much for responding. Now on to what you said, I did think of that. But back when I used to watch streams on twitch, I would watch on medium or low because I would just listen to the streams as background noise and nothing would happen. I could listen to the streams and play hearthstone for 2 hours straight and nothing would go wrong. Is livestreamer different? Something weird did happen last night where I got an error and it said couldn't load segment, but my computer was doing something in the back round and was being weird for a little while (I think avast was updating and was hogging my internet, this did not happen all the other times my problem persisted so it should have no inlfuence) and then everything fixed itself about 20 minutes later.

Assuming you are correct, is there any way I could confirm it? Whenever I do a ping test, no packets get lost or anything like that. I am sure if I call my isp, they won't be able to help me at all so I am at a loss on what to do.
